server/api/crud/model_monitoring/model_endpoints.py

```diff
@@ -547,6 +554,58 @@ def delete_model_endpoints_resources(project_name: str):
             # store not found, there is nothing to delete
             pass
 
+        # Delete model monitoring stream resources
+        self._delete_model_monitoring_stream_resources(
+            project_name=project_name,
+            db_session=db_session,
+            model_monitoring_applications=model_monitoring_applications,
+            stream_paths=stream_paths,
+        )
+
+    @staticmethod
+    def _delete_model_monitoring_stream_resources(
+        project_name: str,
+        db_session: sqlalchemy.orm.Session,
+        model_monitoring_applications: typing.Optional[list[str]] = None,
+        stream_paths: typing.Optional[list[str]] = None,
+    ) -> None:
+        """
+        Delete model monitoring stream resources.
+
+        :param project_name:                  The name of the project.
+        :param db_session:                    A session that manages the current dialog with the database.
+        :param model_monitoring_applications: A list of model monitoring applications that their resources should
+                                              be deleted.
+        :param stream_paths:                  A list of stream paths to delete. If using Kafka, the stream path
+                                              represents the related topic.
+        """
+
+        model_monitoring_access_key = None
+
+        if stream_paths[0].startswith("v3io"):
+            # Generate V3IO Access Key
+            model_monitoring_access_key = (
+                server.api.api.endpoints.nuclio.process_model_monitoring_secret(
+                    db_session,
+                    project_name,
+                    mlrun.common.schemas.model_monitoring.ProjectSecretKeys.ACCESS_KEY,
+                )
+            )
+
+        # Add the writer and monitoring stream to the application streams list
+        model_monitoring_applications.append(
```

Comment:
If `model_monitoring_applications` is `None`, you can't call `append` on it

Reply:
good catch

Reply:
No change